tests/cli/test_cli.py

```diff
@@ -743,12 +745,11 @@ def test_upload1(ctx):
     )  # Directory listing with 2 headers lines and one file
 
     # Upload a file that exceeds the disk quota
-    username = _run_command([cl, 'uinfo', '-f', 'user_name'])
-    prev_dq = _run_command([cl, 'uinfo', username, '-f', 'disk']).split(' ')[2]
-    _run_command([cl, 'uedit', username, '--disk-quota', '2'])
+    prev_dq = _run_command([cl, 'uinfo', 'codalab', '-f', 'disk']).split(' ')[2]
```

Comment:
we don't need this line anymore

Reply:
would I get the value through `ctx.disk_quota`?

Reply:
We won't be using this variable if we remove the line below.

Reply:
ok, removed the variable